Closes [Term Entry] PHP String Functions chr() #3220 (#3481)
Codecademy/docs: content/php/concepts/string-functions/terms/chr/chr.md

---
Title: 'chr()'
Description: 'Returns a single-character string based on the ASCII value provided as an argument.'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Strings'
  - 'Functions'
CatalogContent:
  - 'learn-php'
  - 'paths/computer-science'
---

The **`chr()`** function in PHP is used to convert an ASCII value into a single-character string.

## Syntax

```pseudo
chr($ascii_value)
```

> It accepts an ASCII value as a parameter and returns a string representing a character from the specified ASCII value.

Reviewer: ```suggestion
It accepts an ASCII value as the argument and returns a string representing the associated character.
```

The ASCII value can be specified in decimal, octal, or hex values.

> - Octal values are defined by a leading 0.
> - Hex values are defined by a leading 0x.

Reviewer: ```suggestion
- Octal values are defined by a leading `0`.
- Hex values are defined by a leading `0x`.
```

## Example

> The example below demonstrate the `chr()` function when different ASCII are passed but their equivalent character is same:

Reviewer: ```suggestion
The example below demonstrate the `chr()` function when different ASCII values are passed but their equivalent character is same:
```
